packages/@angular/cli/utilities/stats.ts

```diff
@@ -41,7 +39,18 @@ export function statsToString(json: any, statsConfig: any) {
         .join('');
 
       return `chunk {${y(chunk.id)}} ${g(files)}${names}${size} ${initial}${flags}`;
-    }).join('\n')}
+    }).join('\n');
+
+  const unchangedChunkNumber = json.chunks.filter((chunk: any) => !chunk.rendered).length;
+  const unchangedChunksMessage = unchangedChunkNumber > 0
+    ? `${unchangedChunkNumber} unchanged chunks`
+    : '';
+
+  return rs(stripIndents`
+    Date: ${w(new Date().toISOString())}
+    Hash: ${w(json.hash)}
+    Time: ${w('' + json.time)}ms
+    ${changedChunksStats ? changedChunksStats + '\n' : ''}${unchangedChunksMessage}
```

Comment:
Doesn't seem possible for `changedChunksStats` to be falsy it's always an array.